yarn-project/deploy_npm.sh

```diff
@@ -99,3 +99,4 @@ deploy_package p2p
 deploy_package world-state
 deploy_package sequencer-client
 deploy_package aztec-node
+deploy_package kv-store
```

Comment:
Should this be deployed earlier up the dependency chain?

Reply:
Yea should be after `foundation`. Couple other packages use it (aztec-node including)

Reply:
Where it's at now (line 92) it precedes every package that depends on it.